stable/weblate/values.yaml

```
@@ -79,14 +79,17 @@ service:
   port: 80
 
 ingress:
-  enabled: false
+  enabled: true
+  className: ""
   annotations:
     {}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
   hosts:
     - host: chart-example.local
-      paths: []
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
```

```suggestion
          pathType: Prefix
```

Prefix feels like a more stable default to me, it's more strict in what it should represent so the implementation across controllers should be more consistent.

---

done

---

ImeplementationSpecific gives more freedom as its matching is up to the IngressClass. Implementations can treat this as a separate pathType or treat it identically to Prefix or Exact path types.

---

I know, which is why I thought using `Prefix` would be more consistent. But it's not a hill that I'll die on, both is ok. It's just a default anyway.